cmd/pomerium/options.go

```diff
@@ -39,6 +42,7 @@ type Options struct {
 
 var defaultOptions = &Options{
 	Debug:    false,
+	LogLevel: "debug",
```

Comment:
@desimone should the default be debug? I would have expected a default to info or equivalent, no?

Reply:
I could go either way on this which is part of why I wanted to break it out into a user configurable field. Here's my thinking. 

Right now, the DEBUG is basically a switch for people who want to view individual requests details which includes not only the typical request details (time to serve, request id, HTTP verb, url, etc) but also the attached user's details.  I personally find the details especially useful for auditing purposes and was tempted to move it up to INFO but knew others wouldn't want those details. 

I also think it's a good default option because for new users having any issues, I imagine it would be fairly challenging to investigate any problems without the DEBUG switch which adds another step.  When users are ready to deploy pomerium in production, it's probably safe to assume they've become comfortable enough to set the log level they feel best fits their usage but at first, if they hit some issue, I wouldn't want the added frustration of finding the verbose switch.

Thoughts?

Reply:
Interesting, glad I raised the question, because personally, I don't really care to have all the http details (I already have nginx-ingress for that) BUT I am mainly interested in the auditing info such as user details.
I would even go with the user details in INFO: it is about the main business of pomerium (and I guess in the long term serious auditing could be one of its feature :).

Also I think the argument about setting info while in production and having debug by default is totally sound yes!

Reply:
The trick is associating those user details with a given request so it's hard to separate the two (even though, like you said, the many users will be put pomerium behind a 3rd party ingress). Without the additional HTTP context(`JoeUser with admins@company.com access /intranet/hr/payroll at 3:20 from IP x.x.x.x`) having user details isn't very useful for auditing purposes which absolutely is a primary usecase for the product. (Fun fact and a tangible example, if you read the Uber vs Waymo lawsuit,  BeyondCorp played a major role in helping to audit the alleged IP theft...)

Reply:
I'm going to merge but I like the idea of having audit information demarcated differently, maybe by log level, maybe by another mechanism.

Reply:
> The trick is associating those user details with a given request so it's hard to separate the two

Yes, I guess you still need them

> I'm going to merge but I like the idea of having audit information demarcated differently, maybe by log level, maybe by another mechanism.

This will be the subject of another, more security/audit-oriented release maybe :)